content/docs/user-guide/plotting-in-offline-environments.md

```diff
@@ -0,0 +1,80 @@
+# Plotting in offline environments
+
+There may be cases when the user does not have an Internet connection for
+various reasons. `dvc plots` - generated HTML will not render properly in these
+cases. This guide shows how to fix plotting in offline environments.
```

Comment:
TBH I didn't review the new guide until now since I thought the plan was to extract it from this PR. In general it needs some syntax check, simplification, summarizing, linking, etc., I think e.g.

```suggestion
If there' no Internet connection, the HTML file generated by `dvc plots` will not render properly.
This guide shows how to display plots when offline.
```

Reply:
TBH I also wouldn't introduce the new guide yet (there's #2572 to figure that out) but if others approve, then OK but let's just rewrite it per the guidelines above based on this text that @pared provided? Thanks

Reply:
[Here](https://github.com/iterative/dvc.org/pull/2472#issuecomment-865811690) I said

> I'll delete the UG file if everything looks all right. 

I don't intend to keep that guide.

Reply:
> I'll delete the UG file if everything looks all right.

Ah OK cool please do then 🙂